docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,56 @@
+The first step to troubleshooting a ecChronos issue is to use error messages, metrics and monitoring information to identify if the issue lies with the clients or the server and if it does lie with the server find the problematic nodes in the Cassandra cluster. The goal is to determine if this is a systemic issue (e.g. a problem pattern that affects the entire cluster) or isolated to a subset of instances (e.g. neighbors holding a lock or even a single node with bad hardware).
+
+## Client Logs and Errors
+During repair operations in a cassandra instance, triggered by ecChronos, it can return erros logs, like below:
+
+|Error                                                                   | Possible cause                                                                                     |
+|------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+|"Unexpected interrupt while trying to connect to Cassandra"             | Cassandra Node Overloaded, Bad Configuration on Installing, ecChronos Tables does not exists, etc. |
+|"Unable to register watch service, configuration refresh will not work" | Path Directory does not exists, File Permission disagred with request, etc.                        |
+|"Unable to get maxRepaired for {<table_reference>}"                     | Cassandra Node Overloaded, Instance Not Found, Table does not Exists Anymore, etc.                |
+
+*Futhermore, one common problem is when repair is taking too much time/is overdue. This can be caused by too many vnodes, nodes going down while holding a lock and Cassandra nodes being down.*
+
+## Metrics
+If you have Cassandra metrics reporting to a centralized location such as Graphite or Grafana you can typically use those to narrow down the problem. But ecChronos can be monitored, as described in the [repair monitoring documentation](REPAIR_MONITORING.md). Using metrics, it can be possible to identify some possible problems in the cluster and evaluete some basic things, like overload, network issues, cassandra rejections, etc.
+
+## Use ECCTOOL
+ecctool is a command line utility which can be used to perform actions towards a local ecChronos instance. The actions are implemented in form of subcommands with arguments. All visualization is displayed in form of human-readable tables.
+
+To have more information about ecctool you can check the [ecctool examples](ECCTOOL_EXAMPLES.md) and [ecctool documentation](autogenerated/ECCTOOL.md)
+
+## Cassandra Troubleshooting
+As ecChronos is a client that sends requests to Cassandra to carry out repairs, sometimes it may happen that the Cassandra node does not respond for some reason, so Cassandra itselt have a great set of tools an strategys to troubleshoot an issue, which can be found in the [Cassandra Troubleshooting Documentation](https://cassandra.apache.org/doc/latest/cassandra/troubleshooting/index.html).
```

Comment:
itselt -> itself

"an strategys" -> "and strategies"

The entire sentence is way too long with too many commas

Reply:
This is still left

Reply:
I've removed part of the text and some commas, but I don't know if a shorter text than this could be descriptive.